src/Aquifer.AI/OpenAiTranslationService.cs

```diff
@@ -37,6 +37,7 @@ public class OpenAiOptions
 public sealed class OpenAiTranslationOptions
 {
     public required string DefaultLanguageSpecificPromptAppendixFormatString { get; init; }
+    public required string HtmlAquiferizationBasePrompt { get; init; }
```

Comment:
Now that this service does Aquiferization I'm not sold on the existing name for the service and the options but I can't think of anything better that isn't really long or more confusing (`TransmutationService`? 😆)

Reply:
Yeah I'd briefly thought about renaming or going through everywhere in the message subscriber and putting `translation/aquiferization` for comments and such. If you squint and view aquiferization as a subset of translation (English -> global English) then it feels like the naming still works. But still open to doing a rename if there's something better.